src/main/java/com/snowflake/kafka/connector/internal/streaming/TopicPartitionChannel.java

```diff
@@ -1320,6 +1322,16 @@ public Pair<List<Map<String, Object>>, List<Long>> getData() {
                 kafkaSinkRecord.kafkaOffset(),
                 kafkaSinkRecord.topic());
             kafkaRecordErrorReporter.reportError(kafkaSinkRecord, e);
```

Comment:
shouldn't also records, which raised JsonProcessingException end up in the DLQ? 
i mean - we' ve consumed exception ,reported an error and the message is gone.

Reply:
but they end up, we call kafkaRecordErrorReporter.reportError(kafkaSinkRecord, e);
What do u mean?

Reply:
+1 they are in DLQ.